src/ensembl/src/content/app/entity-viewer/shared/helpers/entity-helpers.ts

```diff
@@ -59,6 +59,15 @@ export type IsProteinCodingTranscriptParam = {
   }>;
 };
 
+type ProteinCodingGeneProps = {
```

Comment:
`ProteinCodingGeneProps` -> `IsProteinCodingGeneParam`? (similar to the type name of the parameter of `isProteinCodingTranscript`)?